apps/src/weblab/brambleHost.js

```diff
@@ -0,0 +1,195 @@
+/* global requirejs */
+
+/**
+ * JS to communicate between Bramble and Code Studio
+ */
+
+(function () {
```

Comment:
no need for this all to be in an anon function now that it's going through webpack